Examples/Algorithms/TrackFitting/src/GlobalChiSquareFitterFunction.cpp

```diff
@@ -68,6 +68,9 @@ struct GlobalChiSquareFitterFunctionImpl final : public TrackFitterFunction {
   bool multipleScattering = false;
   bool energyLoss = false;
   Acts::FreeToBoundCorrection freeToBoundCorrection;
+  std::size_t nUpdateMax = 5;
+  bool zeroField = false;
+  double relChi2changeCutOff = 1e-7;
```

Comment:
Let's revert it in this PR, since clang dislikes (constructor does not initialize these fields). Let's refactor in the future the defaulting of the fitters in general, that we are not mixing the default values.